bindings/python/release.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -ex
+
+# This script should be run on macOS to create the following distributions:
+# pymongocrypt-<version>.tar.gz
+# pymongocrypt-<version>-py2.py3-none-manylinux2010_x86_64.whl
+# pymongocrypt-<version>-py2.py3-none-macosx_10_9_x86_64.whl
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Build the source dist first
+rm -rf build
+python3.7 setup.py sdist
+
+# The libmongocrypt git revision release to embed in our wheels.
+REVISION=latest
+
+# Build the manylinux2010 wheel
+rm -rf ./libmongocrypt
+curl -O https://s3.amazonaws.com/mciuploads/libmongocrypt/rhel-62-64-bit/master/${REVISION}/libmongocrypt.tar.gz
```

Comment:
Why do you need to do this? Isn't the next line just going to overwrite it?

Reply:
Whoops, that's some leftover cruft. Fixed.